src/_kw

```diff
@@ -18,10 +18,14 @@ _kw()
     # Set the context for the command
     curcontext="${curcontext%:*:*}:kw-${command}"
     # Narrow the range of words we are looking at to exclude 'kw'
-    (( CURRENT-- ))
+    ((CURRENT--))
     shift words
-    # Run the completion for the command
-    _kw_${command}
+    # Run the completion for the command if it exists
+    if [[ $(type "_kw_${command}") =~ 'not found' ]]; then
+      _nothing
+    else
+      _kw_${command}
```

Comment:
This eval automatically? Don't you need `eval`?

Reply:
It does eval automatically, but, as we are doing an expansion, I think it is better to use `eval` 👍